source/includes/_resource.collections.md

```diff
@@ -547,7 +557,7 @@ GET /collections/:id/items
 ```
 
 ```ruby
-Vhx::CollectionItem.list()
+Vhx::Item.all()
```

Comment:
Node is representing like this: 
```
vhx.collections.items({
  collection: 'https://api.vhx.tv/collections/1',
  page: 1
});
```
We could represent in ruby as `Vhx::Collection::Item` or also only be able to get items like this `collection.items`

Any preferences?